src/components/admin/settings/YearCourseDeleteModal.tsx

```diff
@@ -31,7 +49,20 @@ export const YearCourseDeleteModal = (props: Props) => {
         <ModalHeader>Usuwanie kierunku</ModalHeader>
         <ModalCloseButton />
         <ModalBody>
-          <Text>Czy na pewno chcesz usunąć ten kierunek?</Text>
+          <Text>
+            Usunięcie kierunku spowoduje także usunięcie {query.data ? query.data.length : ''} przedmiotów, group i
```

Comment:
If there are no classes assigned to the course we want to delete then it doesn't make sense to view this exact notification. We should determine if there are any subjects assigned and if so then show this notification, if not then the previous one.